packages/components/src/internal/components/permissions/PermissionAssignments.tsx

```diff
@@ -72,6 +72,7 @@ export const PermissionAssignments: FC<PermissionAssignmentsProps> = memo(props
     const [groupMembership, setGroupMembership] = useState<GroupMembership>();
     const [error, setError] = useState<string>();
     const [submitting, setSubmitting] = useState<boolean>(false);
+    const canInherit = getServerContext().project.rootId !== containerId;
```

Comment:
the `project` variable exists a couple of lines down at ~78 from the `userServerContext()` hook, any reason you didn't use that one instead of getting it from `getServerContext()`?